config/environments/production.rb

```diff
@@ -74,4 +74,7 @@
   config.action_mailer.default_url_options = {
     host: (ENV['EMAIL_DOMAIN'] || 'chillingeffects.org')
   }
+  config.action_mailer.default_url_options = { host: (ENV['EMAIL_DOMAIN'] || 'chillingeffects.org') }
```

Comment:
these should be updated to lumendatabase.org but perhaps a more through audit should be made as a separate task to fix them all at once